service.go

```diff
@@ -218,7 +221,7 @@ func (svc *Service) createResponse(initialEvent *nostr.Event, content interface{
 	}
 	resp := &nostr.Event{
 		PubKey:    svc.cfg.IdentityPubkey,
-		CreatedAt: time.Now(),
+		CreatedAt: nostr.Now(),
```

Comment:
what is nostr.Now() ? 🤔

Reply:
https://github.com/nbd-wtf/go-nostr/blob/HEAD/timestamp.go